docs_app/content/guide/installation.md

```diff
@@ -26,33 +26,19 @@ To import only what you need, please {@link guide/importing#es6-via-npm check ou
 
 ## CommonJS via npm
 
-If you receive an error like error TS2304: Cannot find name 'Promise' or error TS2304: Cannot find name
-'Iterable' when using RxJS you may need to install a supplemental set of typings.
+If you receive an error like `error TS2304: Cannot find name 'Promise'` or `error TS2304: Cannot find name
+'Iterable'` when using RxJS you may need to install a supplemental set of typings.
 
 1.  For typings users:
 
-```shell
-typings install es6-shim --ambient
-```
+  ```shell
+  typings install es6-shim --ambient
+  ```
```

Comment:
This fixes indentation:

![Screenshot 2023-03-09 at 12 28 52](https://user-images.githubusercontent.com/28087049/224010712-6ed97ce6-eedb-43a7-a494-ac7d2df757b5.png)

Vs. how it was:

![Screenshot 2023-03-09 at 12 28 41](https://user-images.githubusercontent.com/28087049/224010672-f5b3acf2-70d2-42ff-be78-88ab43601505.png)

Reply:
It seems that the indent of the code should be equal to the indent of the text.

Reply:
> It seems that the indent of the code should be equal to the indent of the text.

It is now, isn't it? 🙂

Reply:
Thnx, fixed!

Reply:
Shouldn't have them. Please refresh the page, hard refresh may be needed.

Reply:
Now? 😄

Reply:
![Perfection!](https://user-images.githubusercontent.com/10235949/224036863-5913ca10-cfc0-47ca-9a64-1a4c0815d44c.png)

Reply:
Thanks for teaching me how Prettier works. I wished so many times in my life projects don't use it 😱

Reply:
Well, here the problem is rather not in Prettier, but in the ambiguity of Markdown. I searched and saw that the results are really different for different parsers if the indent does not match the position of the first line, and this applies not only to blocks of code, but also to other elements, such as paragraphs. The formatted MD is processed correctly.